pom.xml

```diff
@@ -683,13 +684,6 @@
     </profile>
 
     <profile>
-      <id>scalding</id>
-      <modules>
-        <module>scalding</module>
-      </modules>
-    </profile>
```

Comment:
Remove scalding profile and includes in the module list result [release script](https://github.com/apache/incubator-zeppelin/blob/master/dev/create_release.sh) creates a binary package with scalding interpreter included. Therefore, we need to take care few more things for binary package release.

One thing is Zeppelin want to avoid build binary package for release with 3rd party repository but scalding interpreter needs two 3rd party repositories (conjars.org/repo, maven.twttr.com).
Another thing updating LICENSE for binary package, while scalding interpreter brings new dependency libraries into binary package.  `mvn -DskipTests -pl 'zeppelin-interpreter,scalding' package dependency:tree` will give detailed list of dependencies (and transitive dependency).

Here's partial list of dependencies of scalding interpreter.
```
[INFO] +- com.twitter:scalding-core_2.11:jar:0.16.1-RC1:compile
[INFO] |  +- com.twitter:scalding-serialization_2.11:jar:0.16.1-RC1:compile
[INFO] |  +- com.twitter:maple:jar:0.16.1-RC1:compile
[INFO] |  +- cascading:cascading-core:jar:2.6.1:compile
[INFO] |  |  +- riffle:riffle:jar:0.1-dev:compile
[INFO] |  |  +- thirdparty:jgrapht-jdk1.6:jar:0.8.1:compile
[INFO] |  |  \- org.codehaus.janino:janino:jar:2.7.5:compile
[INFO] |  |     \- org.codehaus.janino:commons-compiler:jar:2.7.5:compile
[INFO] |  +- cascading:cascading-hadoop:jar:2.6.1:compile
[INFO] |  +- cascading:cascading-local:jar:2.6.1:compile
[INFO] |  |  \- com.google.guava:guava:jar:15.0:compile
[INFO] |  +- com.twitter:chill-hadoop:jar:0.7.3:compile
[INFO] |  |  \- com.esotericsoftware.kryo:kryo:jar:2.21:compile
[INFO] |  |     +- com.esotericsoftware.reflectasm:reflectasm:jar:shaded:1.07:compile
[INFO] |  |     |  \- org.ow2.asm:asm:jar:4.0:compile
[INFO] |  |     +- com.esotericsoftware.minlog:minlog:jar:1.2:compile
[INFO] |  |     \- org.objenesis:objenesis:jar:1.2:compile
[INFO] |  +- com.twitter:chill-java:jar:0.7.3:compile
[INFO] |  +- com.twitter:chill-bijection_2.11:jar:0.7.3:compile
[INFO] |  +- com.twitter:algebird-core_2.11:jar:0.12.0:compile
[INFO] |  |  \- com.googlecode.javaewah:JavaEWAH:jar:0.6.6:compile
[INFO] |  +- com.twitter:bijection-core_2.11:jar:0.9.1:compile
[INFO] |  +- com.twitter:bijection-macros_2.11:jar:0.9.1:compile
[INFO] |  +- com.twitter:chill_2.11:jar:0.7.3:compile
[INFO] |  \- com.twitter:chill-algebird_2.11:jar:0.7.3:compile
[INFO] +- com.twitter:scalding-args_2.11:jar:0.16.1-RC1:compile
[INFO] +- com.twitter:scalding-date_2.11:jar:0.16.1-RC1:compile
[INFO] +- com.twitter:scalding-commons_2.11:jar:0.16.1-RC1:compile
[INFO] |  +- com.google.protobuf:protobuf-java:jar:2.4.1:compile
[INFO] |  +- com.twitter.elephantbird:elephant-bird-cascading2:jar:4.8:compile
[INFO] |  +- com.twitter.elephantbird:elephant-bird-core:jar:4.8:compile
[INFO] |  |  +- com.twitter.elephantbird:elephant-bird-hadoop-compat:jar:4.8:compile
[INFO] |  |  \- com.googlecode.json-simple:json-simple:jar:1.1:compile
[INFO] |  \- com.hadoop.gplcompression:hadoop-lzo:jar:0.4.19:compile
[INFO] |     \- commons-logging:commons-logging:jar:1.1.1:compile
[INFO] +- com.twitter:scalding-avro_2.11:jar:0.16.1-RC1:compile
[INFO] |  +- cascading.avro:avro-scheme:jar:2.1.2:compile
[INFO] |  |  +- org.apache.avro:avro-mapred:jar:1.7.4:compile
[INFO] |  |  |  +- org.apache.avro:avro-ipc:jar:1.7.4:compile
[INFO] |  |  |  |  +- org.mortbay.jetty:jetty:jar:6.1.26:compile
[INFO] |  |  |  |  +- org.apache.velocity:velocity:jar:1.7:compile
[INFO] |  |  |  |  \- org.mortbay.jetty:servlet-api:jar:2.5-20081211:compile
[INFO] |  |  |  \- org.apache.avro:avro-ipc:jar:tests:1.7.4:compile
[INFO] |  |  \- cascading:cascading-xml:jar:2.1.6:compile
[INFO] |  |     \- org.ccil.cowan.tagsoup:tagsoup:jar:1.2:compile
[INFO] |  \- org.apache.avro:avro:jar:1.7.4:compile
[INFO] |     +- org.codehaus.jackson:jackson-core-asl:jar:1.8.8:compile
[INFO] |     +- org.codehaus.jackson:jackson-mapper-asl:jar:1.8.8:compile
[INFO] |     +- com.thoughtworks.paranamer:paranamer:jar:2.3:compile
[INFO] |     +- org.xerial.snappy:snappy-java:jar:1.0.4.1:compile
[INFO] |     \- org.apache.commons:commons-compress:jar:1.4.1:compile
[INFO] |        \- org.tukaani:xz:jar:1.0:compile
[INFO] +- com.twitter:scalding-parquet_2.11:jar:0.16.1-RC1:compile
[INFO] |  +- org.apache.parquet:parquet-column:jar:1.8.1:compile
[INFO] |  |  +- org.apache.parquet:parquet-common:jar:1.8.1:compile
[INFO] |  |  +- org.apache.parquet:parquet-encoding:jar:1.8.1:compile
[INFO] |  |  \- commons-codec:commons-codec:jar:1.5:compile
[INFO] |  +- org.apache.parquet:parquet-hadoop:jar:1.8.1:compile
[INFO] |  |  +- org.apache.parquet:parquet-format:jar:2.3.0-incubating:compile
[INFO] |  |  \- org.apache.parquet:parquet-jackson:jar:1.8.1:compile
[INFO] |  \- org.apache.parquet:parquet-thrift:jar:1.8.1:compile
[INFO] +- com.twitter:scalding-repl_2.11:jar:0.16.1-RC1:compile
[INFO] |  \- jline:jline:jar:2.11:compile
[INFO] +- org.scala-lang:scala-library:jar:2.11.8:compile
[INFO] +- org.scala-lang:scala-compiler:jar:2.11.8:compile
[INFO] |  +- org.scala-lang.modules:scala-xml_2.11:jar:1.0.4:compile
[INFO] |  \- org.scala-lang.modules:scala-parser-combinators_2.11:jar:1.0.4:compile
[INFO] +- org.scala-lang:scala-reflect:jar:2.11.8:compile
```

They need to be addressed in [zeppelin-distribution/src/bin_license/LICENSE](https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-distribution/src/bin_license/LICENSE) file.

I recommend restore scalding profile and create separate issue for removing scalding profile. And we can take care of 3rd party repo and binary package license in the new issue.